stores/migrations.go

```diff
@@ -247,11 +255,23 @@ func initSchema(tx *gorm.DB) error {
 	if err != nil {
 		return fmt.Errorf("failed to init schema: %w", err)
 	}
-	// Change the object_id colum to use case sensitive collation.
+
+	// Change the collation of columns that we need to be case sensitive.
 	if !isSQLite(tx) {
-		return tx.Exec("ALTER TABLE objects MODIFY COLUMN object_id VARCHAR(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_bin;").Error
+		err = tx.Exec("ALTER TABLE objects MODIFY COLUMN object_id VARCHAR(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_bin;").Error
```

Comment:
could be a separate PR (mostly because we need to test it manually)

Reply:
I'm not sure what you mean. The line you commented on is not new code and the one below is necessary for the new buckets table.